projects/npm-tools/packages/npm-scripts/package.json

```diff
@@ -76,6 +77,9 @@
 	"license": "BSD-3-Clause",
 	"main": "index.js",
 	"name": "@liferay/npm-scripts",
+	"devDependencies": {
+		"@clayui/css": "^3.x"
+	},
```

Comment:
We only want `devDependencies` at the root of the repo (I have a few of those to clean up in the recently imported js-themes-toolkit, but the other projects are all free of `devDependencies`).